### DISCUSSION TITLE
Why is my blinds group not working?

### DISCUSSION THREAD

**Discussion by aatifk** [Koenkk/zigbee2mqtt#18150] [Devices]
I again have an issue with my IKEA FYRTUR blinds. I have created a group for the blinds in Z2M and it exposes to Home Assistant just fine, but I can't control the group from HA, only the individual blinds. For instance, clicking the Up/Down arrow in HA UI doesn't do anything at all, but if I open/close them individually, the group reflects that. Did I make a mistake setting them up?

![image](https://github.com/Koenkk/zigbee2mqtt/assets/115054989/14d42b03-ba87-4fa3-be0a-431f9fabecf5)

**Comment by jnxxx** [comment]
I am experiencing the same issue. In my case just with PRAKTLYSING (IKEA E2102).
And scanning through other threads here, it seems it is a general issue.
A group with IKEA bulbs works just fine, but not covers/blinds.

**Response when adding a bulb:**
`Debug 2023-07-18 11:00:38 Received Zigbee message from 'Sengelampe H', type 'read', cluster 'genOnOff', data '["onOff"]' from endpoint 1 with groupID 1`
(where groupID 1 is correct for the group)

**Response when adding PRAKTLYSING:**
`Debug 2023-07-18 11:05:42Received Zigbee message from 'Gardin lille H', type 'readResponse', cluster 'genPowerCfg', data '{"batteryPercentageRemaining":200}' from endpoint 1 with groupID 0`
(where groupID 0 is _not_ correct for the group, probably some default)

Without knowling that much about Zigbee, it seems to me like it did not accept or understand the command to join the group, although Z2M reports it as successful.

I guess either [binding the remote](https://github.com/Koenkk/zigbee2mqtt/discussions/17905) to both blinds or making a group in HA will have to be the solution.

**Comment by majorsl** [comment]
I just got a pair of Third Reality blinds and am having a similar problem, except I can't even add them to a new group.  If you folks try to make a new test group, can you add one of your blinds or does it error out?

**Comment by jnxxx** [comment]
We can add it to the group (also a new one), but nothing happens when we try to control the group.
